src/middleware.ts

```diff
@@ -0,0 +1,29 @@
+import { NextRequest, NextResponse } from 'next/server'
+
+import { DEFAULT_LANGUAGE } from './constants';
+import { stringifyUrlObject } from './utils/routerUtils';
+
+const PUBLIC_FILE = /\.(.*)$/
+/**
+ * Enforce prefix for default locale 'fi'
+ * https://github.com/vercel/next.js/discussions/18419
+ * @param req
+ */
+
+export async function middleware(req: NextRequest) {
+  if (
+    req.nextUrl.pathname.startsWith('/_next') ||
+    PUBLIC_FILE.test(req.nextUrl.pathname) ||
+    req.nextUrl.pathname.includes('/api/')
```

Comment:
Could these paths be configurable in AppConfig. Or at least in a clean constant array.

Reply:
The `/api` comes from `pages/api` folder. It's not configurable in that sense. If you rename the `pages/api` folder, then you should remember to change this too.... or healthcheck will fail. I can try to clarify the code a bit.